<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameListWidget</class>
 <widget class="QWidget" name="GameListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="GamesToolsBar" native="true">
     <layout class="QHBoxLayout" name="GamesToolsBarLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonNoCover">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/resources/icons/white/cover.png</normaloff>
          <normalon>:/icons/resources/icons/white/no_cover.png</normalon>:/icons/resources/icons/white/cover.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="coverSizeIcon">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/icons/resources/icons/white/covers_size.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="layoutTypeComboBox">
        <property name="currentText">
         <string notr="true"/>
        </property>
        <item>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/resources/icons/white/layout_grid3xn.png</normaloff>:/icons/resources/icons/white/layout_grid3xn.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/resources/icons/white/layout_grid3xn.png</normaloff>:/icons/resources/icons/white/layout_grid3xn.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/resources/icons/white/layout_grid3xn.png</normaloff>:/icons/resources/icons/white/layout_grid3xn.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/resources/icons/white/layout_horizontal.png</normaloff>:/icons/resources/icons/white/layout_horizontal.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/resources/icons/white/layout_list.png</normaloff>:/icons/resources/icons/white/layout_list.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="gameListContainerLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="gridSearchHorizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="gridSearchLabel">
       <property name="text">
        <string>search</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="gridSearch"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
